MeretzLib/Apple/iOS/Meretz/Meretz/Meretz.h

```diff
@@ -138,9 +183,20 @@ typedef NS_ENUM(NSInteger, MeretzTaskStatus)
 	// Item consumption over a date range
 	// startDate: date to use for start of the search
 	// endDate: date to use for the end of the search (if nil, current day is assumed)
-	- (MeretzTaskId) vendorConsume: (NSDate *) startDate optional: (NSDate *) endDate;
+	// marks all newly purchased items for the vendor/user/date-range as 'consumed',
+	// and returns all vendor/user items within that date range
+	- (MeretzTaskId) vendorConsumeWithinRange: (NSDate *) startDate optional: (NSDate *) endDate;
+	// Retrieve a list of newly acquired items (read-only)
+	- (MeretzTaskId) vendorConsumeGetNew;
+	// Acknowledge "consumption" of newly acquired items returned from a vendorConsumeGetNew call
+	// input is an array of NSString objects set to the PublicId values of the MeretzItem objects being acknowledged
+	- (MeretzTaskId) vendorConsumeAcknowledge: (NSArray *) meretzItemPublicIdArray;
+	// Retrieve a list of newly acquired items (acknowledging them at the same time)
+	- (MeretzTaskId) vendorConsumeGetNewAndAcknowledge;
+	// obtain results from any of the vendorConsumeXYZ methods
 	- (MeretzVendorConsumeResult *) getVendorConsumeResult: (MeretzTaskId) vendorConsumeTask;
 
+	/* $FUTURE not currently exposed on the back end
```

Comment:
I closed off access to these pending APIs from Bill's pull request (which is still active)